### DISCUSSION TITLE
Create an Admin user assignRole problem

### DISCUSSION THREAD

**Discussion by ehsan1011** [shopperlabs/framework#117] [Q&A] [answered]
Hi,
Thank you for your beautiful work.

By running 

`php artisan shopper:admin`

Result:

```
Creating admin account...
Call to undefined method App\Models\User::assignRole()
User created successfully.
```

And consequently errors in opening the pages:
```
BadMethodCallException
Call to undefined method App\Models\User::hasPermissionTo()
```

**Comment by thearyanahmed** [comment, marked as answer]
Did you extend your User class with '/Shopper/Framework/Models/User/User' instead of the default laravel 'Authenticable'.

**Comment by detygon** [comment]
You should extend the `User` Model as instructed in the [docs](https://laravelshopper.dev/installing#update-existing-files). Then remove the `fillable` and `casts` properties from your model to avoid clashes.
